packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -29,6 +34,48 @@
   private final FlutterWebViewClient flutterWebViewClient;
   private final Handler platformThreadHandler;
 
+  // Verifies that a url opened by `Window.open` has a secure url.
+  private class FlutterWebChromeClient extends WebChromeClient {
```

Comment:
Is it possible to have a single implementation of `WebChromeClient`? Maybe pass to the constructor any parameter that needs to be different between this implementation and https://github.com/flutter/plugins/pull/2991/files#diff-43c39528992b7b93988dcd7d78478281R68

Reply:
I don't think we have a precedent for creating a shared class between plugins. I think the setup and maintenance of that could be more work than having about 10 lines of the same code.

Reply:
nvm. I didn't realize this is fixing two different plugins.